src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const list = document.getElementById('thumbs');
+const bigItem = document.getElementById('largeImg');
+
+list.addEventListener('click', (event) => {
```

Comment:
```suggestion
list.addEventListener('click', (clickEvent) => {
```
![image](https://github.com/mate-academy/js_gallery_DOM/assets/96409563/58310b43-faa8-48a8-ad15-c6900b532051)